src/machinery/WalletState.ts

```diff
@@ -1,10 +1,10 @@
-import type { NanoAddress, NanoTransaction, NanoWallet } from './models';
+import type { NanoAccount, NanoTransaction, NanoWallet } from './models';
 import { walletStore } from '../stores/stores';
 
 export interface WalletState {
   wallet: NanoWallet | undefined;
-  selectedAccount?: NanoAddress;
   transactions?: NanoTransaction[];
+  account?: NanoAccount;
```

Comment:
We now only deal with a "selected account", instead of selected address and having to figure out the account state afterwards. When an account gets selected we pull history, pending blocks and representative for that account.